src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/legalrepresentative/LegalRepresentativeRecordOutOfTimeDecisionCanProceed.java

```diff
@@ -0,0 +1,83 @@
+package uk.gov.hmcts.reform.iacasenotificationsapi.domain.personalisation.legalrepresentative;
+
+import static java.util.Objects.requireNonNull;
+import static uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCaseDefinition.*;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Set;
+import javax.validation.constraints.NotNull;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCase;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCaseDefinition;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.OutOfTimeDecisionType;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.personalisation.EmailNotificationPersonalisation;
+import uk.gov.hmcts.reform.iacasenotificationsapi.infrastructure.CustomerServicesProvider;
+
+@Service
+public class LegalRepresentativeRecordOutOfTimeDecisionCanProceed implements LegalRepresentativeEmailNotificationPersonalisation {
+
+    private final String legalRepresentativeRecordOutOfTimeDecisionCanProceedTemplateId;
+    private final String iaExUiFrontendUrl;
+    private final CustomerServicesProvider customerServicesProvider;
+
+    public LegalRepresentativeRecordOutOfTimeDecisionCanProceed(
+        @NotNull(message = "recordOutOfTimeDecisionCanProceedTemplateId cannot be null")
+        @Value("${govnotify.template.recordOutOfTimeDecision.legalRep.canProceed.email}")
+            String legalRepresentativeRecordOutOfTimeDecisionCanProceedTemplateId,
+        @Value("${iaExUiFrontendUrl}") String iaExUiFrontendUrl,
+        CustomerServicesProvider customerServicesProvider
+    ) {
+        this.legalRepresentativeRecordOutOfTimeDecisionCanProceedTemplateId =
+            legalRepresentativeRecordOutOfTimeDecisionCanProceedTemplateId;
+        this.iaExUiFrontendUrl = iaExUiFrontendUrl;
+        this.customerServicesProvider = customerServicesProvider;
+    }
+
+    @Override
+    public Set<String> getRecipientsList(AsylumCase asylumCase) {
```

Comment:
please remove the override because default implementation from the service should be taken